content/docs/start/data-access.md

```diff
@@ -57,7 +57,7 @@ file that can be saved in the project:
 
 ```dvc
 $ dvc import https://github.com/iterative/dataset-registry \
-             get-started/data/data.xml -o data/data.xml
+             get-started/data.xml -o data/data.xml
```

Comment:
@shcheklein is this intended? Are you planning to update iterative/dataset-registry soon? Thanks

Reply:
nah, that's a bug, thanks @ddobrinskiy !